keps/sig-windows/20190103-windows-node-support.md

```diff
@@ -162,19 +189,19 @@ tolerations:
 - SIG-Windows has high confidence to the stability and reliability of Windows Server containers on Kubernetes
 - 100% green/passing conformance tests that are applicable to Windows (see the Testing Plan section for details on these tests). These tests are adequate, non flaky, and continuously run. The test results are publicly accessible, enabled as part of the release-blocking suite
 - Compatibility will not be broken, either for existing users/clusters/features or for the new features going forward, and we will adhere to the deprecation policy (https://kubernetes.io/docs/reference/using-api/deprecation-policy/).
-- Comprehensive documentation that includes but is not limited to the following sections. Documentation will reside at https://kubernetes.io/docs and will adequately cover end user and admin documentation that describes what the user does and how to use it
+- Comprehensive documentation that includes but is not limited to the following sections. Documentation will reside at https://kubernetes.io/docs and will adequately cover end user and admin documentation that describes what the user does and how to use it. Not all of the documentation will be under the Getting Started Guide for Windows. Part of it will reside in its own sections (like the Group Managed Service Accounts) and part will be in the Contributor development guide (like the instructions on how to build your own source code)
 1. Outline of Windows Server containers on Kubernetes
 2. Getting Started Guide, including Prerequisites
-3. How to deploy Windows nodes in Kubernetes
+3. How to deploy Windows nodes in Kubernetes and where to find the proper binaries (Listed under the changelog for every release. For example https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.13.md#server-binaries)
 4. Overview of Networking on Windows
 5. Links to documentation on how to deploy and use CNI plugins for Windows (example for OVN - https://github.com/openvswitch/ovn-kubernetes/tree/master/contrib)
 6. Links to documentation on how to deploy Windows nodes for public cloud providers or other Kubernetes distributions (example for Rancher - https://rancher.com/docs//rancher/v2.x/en/cluster-provisioning/rke-clusters/windows-clusters/)
 7. How to schedule Windows Server containers, including examples
-8. Advanced: How to use metrics and the Horizontal Pod Autoscaler
-9. Advanced: How to use Group Managed Service Accounts
-10. Advanced: How to use Taints and Tolerations for a heterogeneous compute cluster (Windows + Linux)
-11. Advanced: How to use Hyper-V isolation (not a stable feature yet)
-12. Advanced: How to build Kubernetes for Windows from source
+8. How to use metrics and the Horizontal Pod Autoscaler
+9. How to use Group Managed Service Accounts
+10. How to use Taints and Tolerations for a heterogeneous compute cluster (Windows + Linux)
+11. How to use Hyper-V isolation (not a stable feature yet)
+12. How to build Kubernetes for Windows from source
```

Comment:
This URL right? https://github.com/PatrickLang/kubernetes-windows-dev#setting-up-a-devbuild-environment

Reply:
we will revamp it, but yes that's a good start

Reply:
I think that's only if you need to build k8s on a Windows dev box. On Linux or OSX `make cross KUBE_BUILD_PLATFORMS="windows/amd64"` is all you need.